lib/src/runtime.dart

```diff
@@ -16,6 +16,9 @@ import 'wasmer_api.dart';
 
 part 'runtime.g.dart';
 
+/// The singleton instance of [WasmRuntime].
+final runtime = WasmRuntime._init();
```

Comment:
Can we make this private?

Reply:
Sadly, no. Used in `module.dart`. It's still not "public".